include/cutlass/epilogue/threadblock/epilogue_streamk_with_broadcast.h

```diff
@@ -0,0 +1,1782 @@
+/***************************************************************************************************
```

Comment:
i understand some code duplication is inevitable, but we want to keep it low.

is there any chance to share the code with the streamk epilogue?  e.g. use the streamk epilogue as the base class, then we can call `Base::reduce()`?

can we refactor the vector load function from non streamk broadcast and share between streamk and non streamk version?

Reply:
I might be misunderstanding, but we actually are already doing that already. EpilogueStreamkWithBroadcast inherits from both EpilogueBase and EpilogueStreamkBase:

https://github.com/NVIDIA/cutlass/blob/17cb1fb0bb292da26dd0d0467b02a69d41c5e857/include/cutlass/epilogue/threadblock/epilogue_streamk_with_broadcast.h#L226-L238

and calls EpilogueStreamkBase's reduce:

https://github.com/NVIDIA/cutlass/blob/17cb1fb0bb292da26dd0d0467b02a69d41c5e857/include/cutlass/epilogue/threadblock/epilogue_streamk_with_broadcast.h#L461

So it's effectively only adding vector (and tensor) load/store to the StreamK one.

That said, what we definitely can do is inherit some of the methods from EpilogueWithBroadcast to minimize duplication. I'll get started on that.